src/cai_lab_to_nwb/zaki_2024/zaki_2024_metadata.yaml

```diff
@@ -1,44 +1,48 @@
 NWBFile:
   keywords:
     - hippocampus
-    - memory integration
+    - learning
```

Comment:
Is the paper published? I vaguely remember I got the keywords from the archive..